test/cypress/integration/PageNotFound.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Search User Page', () => {
+	beforeEach(() => {
+		cy.visit('/ThisWillNeverBeAPage')
+		cy.injectAxe()
+	})
+
+	it('should load page', () => {
+		cy.get('h1').should('contain', 'Page Not Found')
```

Comment:
Same comment as other spec file

```suggestion
		cy.get('h4').should('contain', 'Page Not Found')
```